morphia/src/main/java/org/mongodb/morphia/annotations/Index.java

```diff
@@ -105,6 +105,14 @@
     boolean unique() default false;
 
     /**
+     * Removes an existing index, if the options of the existing index with the same name have changed.
+     *
+     * @deprecated use the {@link IndexOptions} found in {@link #options()}
+     */
+    @Deprecated
+    boolean dropOutdated() default false;
```

Comment:
This shouldn't be here.  It doesn't make sense to add something that's deprecrated.  New features should go on the new code only.